crates/bevy_ui/src/render/mod.rs

```diff
@@ -149,6 +150,7 @@ pub struct ExtractedUiNode {
     pub transform: Mat4,
     pub color: Color,
     pub rect: Rect,
+    pub uv_rect: Option<Rect>,
```

Comment:
I responded the same later on in the review when you suggested using the text math for this but here is the issue repeated:  I tried playing with the math the same way that text does but I couldn't understand the math and they also used the scale value to modify the transform.  I couldn't get this to work without the uv_rect because there is a coupling between the uv calculation and position that I think can't be resolved without this new field.

Reply:
Update: You are 100% correct!  Changed!